client/highlevel-rest/build.gradle

```diff
@@ -36,26 +31,30 @@ dependencies {
   testCompile "org.hamcrest:hamcrest-all:${versions.hamcrest}"
 }
 
-dependencyLicenses {
-  // Don't check the client's license. We know it.
-  dependencies = project.configurations.runtime.fileCollection {
-    it.group.startsWith('org.elasticsearch') == false
+task checkNoOtherDependencies {
```

Comment:
This doesn't need to be a task. It can be done at configuration time, see `BuildPlugin.configureConfigurations`. However, I don't see why adding code in the same file that the dependencies are declared is necessary? Why can't this be just a comment next to the two compile dependencies?